src/index.js

```diff
@@ -0,0 +1,16 @@
+import _getWeb3 from "./utils/getWeb3";
+import _Web3Provider from "./provider/Web3Provider";
+import _withContract from "./hocs/withContract";
+import _withWeb3 from "./hocs/withWeb3";
+
+export const getWeb3 = _getWeb3;
+export const Web3Provider = _Web3Provider;
+export const withContract = _withContract;
+export const withWeb3 = _withWeb3;
+
+export default {
```

Comment:
Why the double export?

`export getWeb3 from "./utils/getWeb3";`

Reply:
Was going for the named + default exports but I can't really think of a good use case for the default.